nodejs/week2/src/backend/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const app = express()
+
+
+const meals = require("./routes/meals.js")
+const reservations = require("./routes/reservations.js")
+const reviews = require("./routes/reviews.js")
+
+
+app.get('/', (req ,res, next)=>{
+    res.send('Hello world')
+    next()
+})
+
+app.use((req, res, next)=>{

```

Comment:
It's not a severe mistake, but just remember, that by putting your middleware *after* `app.get` on line 10, you are not adding it to that specific route. If you want the middleware to be enabled for all of your routes, you have to put it before all of them.